### DISCUSSION TITLE
dprint + oxc instead of our own formatter written from scratch

### DISCUSSION THREAD

**Discussion by Boshen** [oxc-project/oxc#3252] [Ideas]
related: https://github.com/dprint/dprint-swc-ext

**Comment by Boshen** [comment]
Close as not planned.

**Reply by Hebilicious** [reply]
May I ask why this is not planned ? Even as a temporary solution until oxc gets its own parser, this would be great to have imo.

I've been using dprint with the biome plugin combined with the markup_fmt and malva plugins, and it works perfectly in a Vue project. This setup is not really popular at the moment, but in a greenfield project it is excellent and  works with vscode. 

I think with a  oxc dprint plugin, people running Vue/Astro/Svelte could use a similar setup as mine and have a great alternative to prettier+eslint powered by oxc. Here's the biome plugin for reference https://github.com/dprint/dprint-plugin-biome
